### PR TITLE
Task/dtserwfour 5 create transfer confirm details information

### PR DESCRIPTION
**Pull request by shanavas123**

### CODE REVIEW

File: transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModel.java [hunk 1]
```diff
@@ -25,6 +25,7 @@
 
 import android.os.Handler;
 import android.text.TextUtils;
+import android.util.Log;
```
Comment (rajkumar-balakrishnasamy) [inline transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModel.java:28]: Please remove the log

File: transferui/src/main/java/com/hyperwallet/android/ui/transfer/view/CreateTransferActivity.java [hunk 2]
```diff
@@ -61,7 +61,6 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_create_transfer);
-
```
Comment (rajkumar-balakrishnasamy) [inline transferui/src/main/java/com/hyperwallet/android/ui/transfer/view/CreateTransferActivity.java:64]: Please remove this changes

File: transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModel.java (hunk 1 above)
Comment (shanavas123) [reply transferui/src/main/java/com/hyperwallet/android/ui/transfer/viewmodel/CreateTransferViewModel.java]: fixed

File: transferui/src/main/java/com/hyperwallet/android/ui/transfer/view/CreateTransferActivity.java (hunk 2 above)
Comment (shanavas123) [reply transferui/src/main/java/com/hyperwallet/android/ui/transfer/view/CreateTransferActivity.java]: removed

**Review by rajkumar-balakrishnasamy — APPROVED** [review]

**Review by vwagh-hw — APPROVED** [review]

**Pull request merged**